torch/csrc/jit/mobile/parse_bytecode.cpp

```diff
@@ -0,0 +1,122 @@
+#include <ATen/core/ivalue.h>
```

Comment:
Curious: is there some high level analysis of how much it takes to pull in the IValue dependency?

Reply:
Good question. I think it should be < 100 KB if we could decouple it. If we don't want to include Tensor it can be much smaller. I'm going to do further analysis when a simple internal driver is ready.